Lesson23-aysel/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
+    <title>Accordion</title>
+</head>
+<body>
+    <div class="accordion-container">
```

Comment:
I cannot see this element on the page, what is it's purpose? From what I see images in the 'container' div act as accordion.

Reply:
I made it for the gif, there are two accordion containers separately, I probably made a mistake in naming them